### PR TITLE
QuickActions refactoring app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx

### PR DESCRIPTION
**Pull request by jyotiarora2610**
## What kind of change does this PR introduce?

Refactors code as a part of hacktoberfest mentioned in #2621.
@Saeris @christianalfoni

## What is the current behavior?

packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx was using Cerebral

## What is the new behavior?

packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx is using Overmind

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

1. Replaced inject and hooksObserver with useOvermind
2. yarn test
3. yarn lint
4. yarn typecheck

## Checklist

- [ ] Documentation
- [ ] Testing 
- [ ] Ready to be merged
      
- [ ] Added myself to contributors table

### CODE REVIEW

File: packages/app/src/app/pages/Live/index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { SubTitle } from 'app/components/SubTitle';
 import { Skeleton } from 'app/components/Skeleton';
 import { Navigation } from 'app/pages/common/Navigation';
 import { SignInButton } from 'app/pages/common/SignInButton';
-import { QuickActions } from 'app/pages/Sandbox/QuickActions';
+import { QuickActions } from 'app/pages/Sandbox/QuickActions/QuickActions';
```
Comment (vilvaathibanpb) [inline packages/app/src/app/pages/Live/index.js]: I guess this change is not required

File: packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
```
Comment (vilvaathibanpb) [inline packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx]: Even if you rename the file, you should have an index.ts file for exporting the component, so you dont have two add extra file name in the other two files of the PR

File: packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx [hunk 3]
```diff
@@ -17,32 +18,35 @@ import {
   Keybindings,
 } from './elements';
 
-class QuickActionsComponent extends React.Component {
+export class QuickActions extends React.Component {
```
Comment (vilvaathibanpb) [inline packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx]: why is it still class based? It should be functional component

**Comment by christianalfoni** [comment]
Hi there @jyotiarora2610 and thanks for helping out!

For this PR to get through we would need the class based component to be refactored into a function component. You can take a look at for example: https://github.com/codesandbox/codesandbox-client/blob/master/packages/app/src/app/pages/Patron/index.tsx as a reference. 

Let us know if you have any questions related to how to convert the methods of the class component into the function component approach 😄

File: packages/app/src/app/pages/Live/index.js (hunk 1 above)
Comment (jyotiarora2610) [reply packages/app/src/app/pages/Live/index.js]: I've done the changes, please check

File: packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx (hunk 2 above)
Comment (jyotiarora2610) [reply packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx]: I've done the changes, please check

File: packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx (hunk 3 above)
Comment (jyotiarora2610) [reply packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx]: I've done the changes, please check

**Comment by jyotiarora2610** [comment]
> Hi there @jyotiarora2610 and thanks for helping out!
> 
> For this PR to get through we would need the class based component to be refactored into a function component. You can take a look at for example: https://github.com/codesandbox/codesandbox-client/blob/master/packages/app/src/app/pages/Patron/index.tsx as a reference.
> 
> Let us know if you have any questions related to how to convert the methods of the class component into the function component approach 😄

Hi @christianalfoni  I've done the required changes, please check

**Pull request merged**